lib/pure/net.nim

```diff
@@ -611,6 +606,9 @@ when defineSsl:
     when not defined(openssl10) and not defined(libressl):
       let sslVersion = getOpenSSLVersion()
       if sslVersion >= 0x010101000 and not sslVersion == 0x020000000:
+        # XXX always false!
```

Comment:
How?

Reply:
because it's parsed as ((not sslVersion) == 0x020000000)